File: ProjectTwo(Python)/models/accounts.py

class Accounts:

    def __init__(self, acct_id=0, user_id=0, account_type="", account_balance=0):
        self.acct_id = acct_id
        self.user_id = user_id
        self.account_type = account_type
        self.account_balance = account_balance

    def __repr__(self):
        return str({
            'acct_id': self.acct_id,
            'user_id': self.user_id,
            'account_type': self.account_type,
            'account_balance': self.account_balance
        })

    def json(self):
        return str({
            'acctId': self.acct_id,
            'userId': self.user_id,
            'accountType': self.account_type,
            'accountBalance': self.account_balance
        })

File: JavaBackend/src/main/java/models/Accounts.java
package models;

public class Accounts {

    private int acctID;
    private int userID;
    private String accountType;
    private double accountBalance;

    public Accounts(){}

    public Accounts(int acctID, int userID, String accountType, double accountBalance){
        this.acctID=acctID;
        this.userID=userID;
        this.accountType=accountType;
        this.accountBalance=accountBalance;
    }

    public int getAcctID() {
        return acctID;
    }

    public void setAcctID(int acctID) {
        this.acctID = acctID;
    }

    public int getUserID() {
        return userID;
    }

    public void setUserID(int userID) {
        this.userID = userID;
    }

    public String getAccountType() {
        return accountType;
    }

    public void setAccountType(String accountType) {
        this.accountType = accountType;
    }

    public double getAccountBalance() {
        return accountBalance;
    }

    public void setAccountBalance(double accountBalance) {
        this.accountBalance = accountBalance;
    }


}
